### PR TITLE
Add Aqara GZCGQ11LM

### PR DESCRIPTION
**Pull request by xfzha**
illuminance & illuminance_lux report correct.
set detection period time requires click the button of the device.

### CODE REVIEW

File: devices/xiaomi.js [hunk 1]
```diff
@@ -1379,4 +1379,20 @@ module.exports = [
...
+    {
+        zigbeeModel: ['lumi.sen_ill.agl01'],
+        model: 'GZCGQ11LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara T1 light intensity sensor',
+        fromZigbee: [fz.battery, fz.illuminance, fz.aqara_opple_report],
+        toZigbee: [tz.GZCGQ11LM_detection_period],
+        meta: {battery: {voltageToPercentage: '3V_2500'}},
+        exposes: [e.battery(), e.battery_voltage(), e.illuminance(), e.illuminance_lux(),
+            exposes.numeric('detection_period', exposes.access.ALL).withValueMin(1).withValueMax(59).withUnit('s')],
```
Comment (Koenkk) [inline devices/xiaomi.js:1391]: What does this `detection_period` do? Can you add a description for it: `.withDescription('....')`

Comment (xfzha) [reply devices/xiaomi.js:1391]: May be the reporting interval.
There is no detailed description of <code>detection_period</code> in the aqara home. I think it means to report a change at an interval. In fact, if the light changes, it is reported according to this interval.

Comment (Koenkk) [reply devices/xiaomi.js:1391]: Then please add that as the description (its better than having nothing)

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**